src/app/interactions/toggle/toggle-sample-4/toggle-sample-4.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, ViewChild } from "@angular/core";
+import { IgxToggleDirective } from "igniteui-angular";
+
+@Component({
+  selector: "app-toggle-sample-4",
+  styleUrls: ["../toggle-samples.scss"],
+  templateUrl: "./toggle-sample-4.component.html"
+})
+export class ToggleSample4Component {
+    @ViewChild(IgxToggleDirective, { static: true }) public toggle: IgxToggleDirective;
+
+    public offsetToggle() {
+        const deltaX: number = 30;
+        const deltaY: number = 30;
```

Comment:
The number type is implicit from the assignment. 
```suggestion
        const deltaX = 30;
        const deltaY = 30;
```